caproto/_constants.py

```diff
@@ -21,12 +21,9 @@
 # Max of ethernet and 802.{2,3} MTU 1500 - 20(IP header) - 8(UDP header)
 SEARCH_MAX_DATAGRAM_BYTES = 1472
 
-# Servers send beacons at some maximum interval. ("Maximum delay between
-# beacons will be limited by server specified parameter, but is commonly 15
-# seconds.") Servers can be presumed dead and dropped by circuits and repeaters
-# after some interval ("usually 30 seconds"). Reference:
-# https://epics.anl.gov/docs/CAproto.html#secVCUnresponsive
-SERVER_MIA_PRESUMED_DEAD = 60  # seconds
```

Comment:
Note about this deletion: We have a standard, spec-specified value for this in `_utils.py` already, `EPICS_CA_CONN_TMO`.